java/java.editor/src/org/netbeans/modules/editor/java/JavaCompletionCollector.java

```diff
@@ -1082,23 +1082,24 @@ private Completion createExecutableItem(CompilationInfo info, ExecutableElement
                 sortParams.append(paramTypeName);
                 if (!inImport && !memberRef) {
                     VariableElement inst = instanceOf(tm, paramName);
-                    insertText.append("${").append(cnt).append(":").append(inst != null ? inst.getSimpleName() : paramName).append("}");
+                    if (insertTextParams) insertText.append("${").append(cnt).append(":").append(inst != null ? inst.getSimpleName() : paramName).append("}");
+                    else if (cnt == 1) insertText.append("$1");
```

Comment:
I assume this is to ensure the cursor is put inbetween `()`.

Reply:
Yes